machine.go

```diff
@@ -367,6 +368,29 @@ func (h *Headscale) ExpireMachine(machine *Machine) {
 	h.db.Save(machine)
 }
 
+// RenameMachine takes a Machine struct and sets the expire field to now.
```

Comment:
This comment seems wrong ;)

Reply:
Oops. Result of pure laziness. Fixing...